js/oai.test.ts

```diff
@@ -0,0 +1,36 @@
+import { buildOpenAIClient } from "./oai";
+
+describe("OAI", () => {
+  test("should use Azure OpenAI", async () => {
```

Comment:
We have somewhat limited tests in our public repos right now because we tend to do a lot of testing in our private repo. I expect this to change over time, but right now, there isn't a great way to incorporate this as an automated test.

However, it's awesome that you write it, because it means if it breaks in the future, we can use it (and update it) to ensure we fix it properly.

Would you mind using `test.skip` (from [here](https://stackoverflow.com/questions/48125230/skip-one-test-in-test-file-jest)) so that it's skipped (but available if someone wants to use it)?